subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -170,11 +170,18 @@ static void update_recv_state_big_cleared(const struct bt_bap_broadcast_sink *si
 	    reason == BT_HCI_ERR_TERM_DUE_TO_MIC_FAIL) {
 		/* Sync failed due to bad broadcast code */
 		mod_src_param.encrypt_state = BT_BAP_BIG_ENC_STATE_BAD_CODE;
+	} else if (reason == BT_HCI_ERR_CONN_TIMEOUT ||
+		   reason == BT_HCI_ERR_LOCALHOST_TERM_CONN ||
+		   reason == BT_HCI_ERR_REMOTE_USER_TERM_CONN) {
+		/* BIS is not synced anymore */
+		mod_src_param.encrypt_state = BT_BAP_BIG_ENC_STATE_NO_ENC;
 	} else {
 		mod_src_param.encrypt_state = recv_state->encrypt_state;
 	}
```

Comment:
Shouldn't the else here just be `mod_src_param.encrypt_state = BT_BAP_BIG_ENC_STATE_NO_ENC;`? 

If `update_recv_state_big_cleared`  is called and it isn't failing due to `BT_HCI_ERR_TERM_DUE_TO_MIC_FAIL` when we are attempting to decrypt, I guess the encrypt state should just always be set to `BT_BAP_BIG_ENC_STATE_NO_ENC`